io.github.realmazharhussain.GdmSettings.yaml

```diff
@@ -39,6 +39,7 @@ modules:
   - name: gdm-settings
     builddir: true
     buildsystem: meson
+    run-tests: true
```

Comment:
This should be a separate commit. This change is not related to "Remove appstream-util".

Reply:
It was development intended to test appdata files.